client/src/components/ContactPage.js

```diff
@@ -1,26 +1,52 @@
-import "../components/About.css"
+import "../components/About.css";
 
 function ContactPage() {
   return (
     <div className="textContainer">
-      <h4>Contact</h4><br></br>
+      <h4>Contact</h4>
+      <br></br>
+      <br></br>
+      <p>HEBESTREIT IMMOBILIEN</p>
+      <br></br>
+      <p>Wildrosenweg 87</p>
+      <br></br>
+      <p>10711 Berlin</p>
       <br></br>
-      <p>HEBESTREIT IMMOBILIEN</p><br></br>
-      <p>Wildrosenweg 87</p><br></br>
-      <p>10711 Berlin</p><br></br>
       <br></br>
```

Comment:
Please use CSS to create distance between elements `<br>` is a bit out of place here. If you had a more text and want to preserve formatting use the [pre tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre)